api/management/api.go

```diff
@@ -228,59 +228,59 @@ func ConvertToKCUser(user UserRepresentation) kc.UserRepresentation {
 // Validate is a validator for UserRepresentation
 func (user UserRepresentation) Validate() error {
 	if user.Id != nil && !matchesRegExp(*user.Id, RegExpID) {
-		return errors.New("Invalid user ID")
+		return errors.New("invalidUserID")
 	}
 
 	if user.Username != nil && !matchesRegExp(*user.Username, RegExpUsername) {
-		return errors.New("Invalid username")
+		return errors.New("invalidUsername")
 	}
 
 	if user.Email != nil && !matchesRegExp(*user.Email, RegExpEmail) {
-		return errors.New("Invalid email")
+		return errors.New("invalidEmail")
 	}
 
 	if user.FirstName != nil && !matchesRegExp(*user.FirstName, RegExpFirstName) {
-		return errors.New("Invalid firstname")
+		return errors.New("invalidFirstname")
 	}
 
 	if user.LastName != nil && !matchesRegExp(*user.LastName, RegExpLastName) {
-		return errors.New("Invalid lastname")
+		return errors.New("invalidLastname")
 	}
 
 	if user.PhoneNumber != nil && !matchesRegExp(*user.PhoneNumber, RegExpPhoneNumber) {
-		return errors.New("Invalid phone number")
+		return errors.New("invalidPhoneNumber")
 	}
 
 	if user.Label != nil && !matchesRegExp(*user.Label, RegExpLabel) {
-		return errors.New("Invalid label")
+		return errors.New("invalidLabel")
 	}
 
 	if user.Gender != nil && !matchesRegExp(*user.Gender, RegExpGender) {
-		return errors.New("Invalid gender")
+		return errors.New("invalidGender")
 	}
 
 	if user.BirthDate != nil && !matchesRegExp(*user.BirthDate, RegExpBirthDate) {
-		return errors.New("Invalid birthdate")
+		return errors.New("invalidBirthdate")
 	}
 
 	if user.Groups != nil {
 		for _, groupID := range *(user.Groups) {
 			if !matchesRegExp(groupID, RegExpID) {
-				return errors.New("Invalid group ID")
+				return errors.New("invalidGroupID")
```

Comment:
Could be invalidParameter.groupID (always the same error code but different parameters)